tooling/api/src/tauri.ts

```diff
@@ -126,10 +127,24 @@ async function invoke<T>(cmd: string, args: InvokeArgs = {}): Promise<T> {
  *
  * @since 1.0.0
  */
-function convertFileSrc(filePath: string, protocol = 'asset'): string {
+function convertFileSrc(
+  filePath: string,
+  protocol = 'asset',
+  http?: boolean
+): string {
   const path = encodeURIComponent(filePath)
+
+  let scheme =
+    window.location.origin === 'http://tauri.localhost' ? 'http' : 'https'
+  if (http === true) {
+    scheme = 'http'
+  }
+  if (http === false) {
+    scheme = 'https'
+  }
```

Comment:
With your code a user can't use `false` to explicitly overwrite it (also no need to check for http's value again in the if since it can only be true at that point). The closest thing to my intentionally stupid code would be to use `typeof boolean` maybe?

To be clear, my intention is to have the user to _always_ be able to overwrite it with true/false even tho i can't think of a concise example where false would be actually needed...

Reply:
right, my bad, the check should be `if (http !== undefined)`